src/components/Roles/Role.tsx

```diff
@@ -0,0 +1,64 @@
+"use client";
+import { useState } from 'react';
+import { motion, AnimatePresence } from 'framer-motion';
+import { IconType } from 'react-icons';
+
+type JobCardProps = {
```

Comment:
Can you add this type to types folder like the others pls

Reply:
Done!